api/src/main/java/org/openmrs/obs/handler/AbstractHandler.java

```diff
@@ -108,16 +89,21 @@ public File getOutputFileToWrite(Obs obs) throws IOException {
 	 * 
 	 * @param filename
 	 * @return the filepart after the period in the given filename
+	 * @deprecated use {@link org.apache.commons.io.FilenameUtils#getExtension(String)} instead.
```

Comment:
can you add since when its `@deprecated since x.x.x use ...`

Reply:
@teleivo Would I add the current platform version here (2.1.3 as far as I can see?) or a future version number?